apps/src/code-studio/pd/workshop_dashboard/IntroPanel.jsx

```diff
@@ -210,13 +211,17 @@ export default class IntroPanel extends React.Component {
               <h4>Step 2: Take attendance</h4>
               <p>
                 After teachers have signed into their Code Studio accounts, use
-                the attendance links below to take attendance.
+                the attendance links below to take attendance. Note: Teachers
```

Comment:
just curious if this wording will be shown in workshops where the participants are facilitators (not teachers)?

Reply:
Good point! @dmcavoy thoughts on updated language?

Reply:
Ah ya good call we can say "Workshop attendees"

Reply:
@dmcavoy I'm actually realizing now that the rest of the text in this element refers to attendees as "Teachers". Should I unify all of them? If so, should I change all of them to "workshop attendees" or "teachers"?

Reply:
Lets us Workshop Attendees for all of it